netlify/resource_site.go

```diff
@@ -27,6 +27,11 @@ func resourceSite() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"domain_aliases": {
+				Type:     schema.TypeList,
```

Comment:
Should this be a set, or does the order matter here?

Reply:
I don't think order really matters in this scenario